### PR TITLE
Updated CssSelector code example to use the new Converter

### PR DESCRIPTION
**Pull request by hiddewie**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 2.8 |
| Fixed tickets | #5782 |

### CODE REVIEW

**Comment by stof** [comment]
your PR description should be updated with the right ``Applies to``

File: components/css_selector.rst [hunk 1]
```diff
@@ -47,11 +47,14 @@ The CssSelector Component
...
 
 The component's only goal is to convert CSS selectors to their XPath
-equivalents::
+equivalents, using :method:`Converter::toXPath() <Symfony\\Component\\CssSelector\\Converter::toXPath>`
 
-    use Symfony\Component\CssSelector\CssSelector;
+.. code-block:: php
 
-    var_dump(CssSelector::toXPath('div.item > h4 > a'));
+    use Symfony\Component\CssSelector\Converter;
```
Comment (stof) [inline components/css_selector.rst]: The class has been renamed to CssSelectorConverter after the initial merge. So this update is wrong

File: components/css_selector.rst [hunk 2]
```diff
@@ -47,11 +47,14 @@ The CssSelector Component
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The component's only goal is to convert CSS selectors to their XPath
-equivalents::
+equivalents, using :method:`Converter::toXPath() <Symfony\\Component\\CssSelector\\Converter::toXPath>`
```
Comment (stof) [inline components/css_selector.rst]: no need to use a special text for the link IMO. Just use ``:method:`Symfony\\Component\\CssSelector\\CssSelectorConverter::toXPath` ``

File: components/css_selector.rst [hunk 3]
```diff
@@ -47,11 +47,14 @@ The CssSelector Component
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The component's only goal is to convert CSS selectors to their XPath
-equivalents::
+equivalents, using :method:`Symfony\\Component\\CssSelector\\CssSelectorConverter::toXPath`
 
-    use Symfony\Component\CssSelector\CssSelector;
+.. code-block:: php
```
Comment (xabbuh) [inline components/css_selector.rst]: I think you should still use the double colon at the end of the previous paragraph and omit the `code-block` directive here.

**Comment by xabbuh** [comment]
:+1:

**Comment by weaverryan** [comment]
Thanks @hiddewie! I added a `versionadded` in sha: 532b0a4fa312bcb0d15c7f3e446102c2cc2d5845

**Pull request merged**